tests/integration/security/authorization_test.go

```diff
@@ -289,11 +289,10 @@ func TestAuthorization_WorkloadSelector(t *testing.T) {
 							newTestCase("[cInNS2]", a, cInNS2, "/policy-ns-root-c", true),
 						},
 					},
-					{ // Send requests to a to vm in ns1.
-						configDst0: util.VMSvc,
-						// TODO(JimmyCYJ): Support multiple VMs in different namespaces for workload selector test,
-						// and set configDst1 to service on VM.
-						configDst1: util.CSvc,
+					{
+						// TODO(JimmyCYJ): Support multiple VMs in different namespaces for workload selector test and set c to service on VM.
+						b: util.VMSvc,
```

Comment:
I don't personally find `b` being a VM less confusing. We use `b` to mean `Service b` in all other cases

Reply:
the log should show it as `vm` instead, the `a`, `b`, ... is now more like a general symbol that happens to be the same as the current k8s workloads.

In the future, the test framework should support multiple VMs, and then we can just have `VMa`, `VMb`, ... and the same test can be run for both k8s and vm with minimal changes.